manager/session.go

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package manager
+
+import (
+	h "github.com/ernestio/ernest-cli/helper"
+	eclient "github.com/ernestio/ernest-go-sdk/client"
+	emodels "github.com/ernestio/ernest-go-sdk/models"
+)
+
+type session struct {
+	cli *eclient.Client
+}
+
+// Get : ..
+func (c *session) Get() *emodels.Session {
```

Comment:
All managers will become on a basic wrapper of ernest-go-sdk with error management, and maybe some special methods. They will be basically calling the same methods on the sdk, plus hanging the command and printing proper errors, they're quite simple.